### DISCUSSION TITLE
这个可以删掉吗

### DISCUSSION THREAD

**Discussion by fq176** [Tokinx/Wing#151] [想法]
![QQ图片1231](https://github.com/user-attachments/assets/d28c036f-518e-40e3-a1c5-465a7e468a5f)
这个能不能删掉啊
![QQ图片2024](https://github.com/user-attachments/assets/7b024630-f12a-4896-b795-5da676c4ebf5)
可以换成这样吗

**Comment by Tokinx** [comment]
未登录的时候就是下面的样子
